### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thelovekesh**
Fixed some common writing errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ code { font-family: 'Fira Code', monospace; }
 - ACE
 - CodeMirror (enable with `font-variant-ligatures: contextual;`)
 
-### Projects using Fira Code
+### Projects using the Fira Code
```
Comment (alexeyten) [inline README.md:170]: Are you sure that we need "the" here?

Comment (thelovekesh) [reply README.md:170]: Fira code is a specific noun here. That's why "the" is needed.

**Comment by RubenVerg** [comment]
You either use "the Fira Code font" or "Fira Code", just like "Rome" and not "the Rome"

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

File: README.md (hunk 1 above)
Comment (alexeyten) [reply README.md:170]: “Fira Code” is a proper name here. You don’t say “the Times New Roman” or “the Comic Sans”

**Comment by wopian** [comment]
The Oxford comma is an American writing form not really used elsewhere.

**Comment by thelovekesh** [comment]
> The Oxford comma is an American writing form not really used elsewhere. Feels kinda like the spammy Hacktoberfest PRs [DO is trying to reduce](https://hacktoberfest.digitalocean.com/hacktoberfest-update).

This repo is not eligible though.:)

**Pull request closed without merging**